python/ray/workers/default_worker.py

```diff
@@ -134,6 +134,11 @@
     default=False,
     action="store_true",
     help="True if Ray debugger is made available externally.")
+parser.add_argument(
+    "--force-load-code-from-local",
+    default=False,
+    action="store_true",
+    help="If true, don't allow execute task by dynamic pickled function.")
```

Comment:
Is `Executing task by dynamic pickled function is not allowed.` sounds a little better?